views/app.html

```diff
@@ -20,7 +20,7 @@
   <link rel="canonical" href="https://webmaker.org/{{ localeInfo.lang }}{{ currentPath }}" />
   <meta property="description" content="{{ gettext("Strips away complexity") }}" />
   <!-- /sharing -->
-
+  <link href='http://fonts.googleapis.com/css?family=Fira+Sans:400,700' rel='stylesheet' type='text/css'>
```

Comment:
hm, no real easy way to do this right now without adding a huge amount of page weight to the whole repo. can we file as a follow up?